test/fixtures/test-runner/esm/loader.mjs

```diff
@@ -0,0 +1,7 @@
+export function resolve(specifier, context, next) {
```

Comment:
I believe a fixture for a loader that only calls `next` already exists. (I wanna say it's call "passthru"—it's too cumbersome to check from my mobile)